README.md

```diff
@@ -192,6 +193,7 @@ You can then apply the Jetstack Secure agent configuration to your cluster:
 
 ```sh
 sed '/namespace:/d' agent-config.yaml | kubectl -n $NAMESPACE apply -f-
+kubectl -n $NAMESPACE scale --replicas=1 $(kubectl -n $NAMESPACE get deploy -oname | grep preflight)
 kubectl -n $NAMESPACE rollout restart $(kubectl -n $NAMESPACE get deploy -oname | grep preflight)
```

Comment:
Shouldn't need the rollout restart, right?

And maybe this could be simplified to `kubectl scale deploy --replicas=1 --selector=kubernetes.io/app=preflight`

Reply:
Good idea. I'll use the selector `app.kubernetes.io/component=preflight` then:

```sh
% k -n test-1 get deploy preflight -oyaml
kind: Deployment
metadata:
  labels:
    app.kubernetes.io/component: preflight                                  # 🔰 this one
    app.kubernetes.io/instance: test-1 
    app.kubernetes.io/managed-by: Helm                                    
    app.kubernetes.io/name: jetstack-secure-gcm
    app.kubernetes.io/version: v0.1.27
    helm.sh/chart: preflight-0.1.0
  name: preflight
  namespace: test-1
```

Reply:
If often found myself in the need of fixing the `agent-config`, and `scale --replicas=1` alone won't re-create the agent pods. I propose to split the above commands into two separate commands:

---

You will now be able to "activate" the Preflight deployment:

```sh
kubectl -n $NAMESPACE scale deploy --replicas=1 --selector=app.kubernetes.io/component=preflight
```

You can then apply the Jetstack Secure agent configuration to your cluster:

```sh
sed '/namespace:/d' agent-config.yaml | kubectl -n $NAMESPACE apply -f-
kubectl -n $NAMESPACE rollout restart $(kubectl -n $NAMESPACE get deploy -oname | grep preflight)
```